tic-tac-toe/src/tic_tac_toe/board.clj

```diff
@@ -0,0 +1,79 @@
+(ns tic-tac-toe.board
+  (:require [tic-tac-toe.marks :refer :all]))
+
+(def dimension 3)
+
+(defn- one-based [value]
+  (+ value 1))
+
+(defn- three-matching-player-symbols? [row]
+  (let [[one two three] row]
+    (and (= one two three) (not(= one nil)))))
+
+(defn- winning-row? [rows]
+  (some true? (map
+                (fn[row] (three-matching-player-symbols? row))
```

Comment:
Hmm.. I don't see any destructuring going on in the anonymous function on line 15. What I had in mind was something like this:
```clojure
(defn- winning-row? [rows]
  (some true? (map three-matching-player-symbols? rows)))
```

Is that what you tried?